File: Functions_2/Ex01.py
#defining function
def max_of_three(x,y,z):
 return max(x,y,z)
#taking input from user
num1 = float(input('Enter first number : '))
num2 = float(input('Enter second number : '))
num3 = float(input('Enter third number : '))
print('The maximum of the three numbers you entered is {}'.format(max_of_three(num1,num2,num3)))

File: Functions_2/ex01.cpp
#include <stdio.h>

int a, b, c;

void MAX(int d, int e, int f){
	int g;
	if (d > e && d > f){
		g = d;
	}
	else if(e > d && e > f){
		g = e;
	}
	else{
		g = f;
	}
	printf("The greater number is %d", g);
}

int main(){
	printf("Input the three numbers\n");
	scanf("%d", &a);
	scanf("%d", &b);
	scanf("%d", &c);
	MAX(a,b,c);
	
	return 0;
}
